packages/teleport-code-generator/src/types.ts

```diff
@@ -49,6 +49,7 @@ export enum ProjectType {
   PREACT = 'Preact',
   STENCIL = 'Stencil',
   ANGULAR = 'Angular',
+  PREACT_CODESANDBOX = 'PreactCodesandbox',
```

Comment:
can we keep this out of ProjectType? and just decide on the template based on the publisher type? ProjectType is used by other projects to display all available outputs

Reply:
Yeah, feels good that way. But I have a doubt if the user tries to download which code we should let them to? Since we will have two different codebases.

Reply:
let's use the default one and the codesandbox one is specific for the codesandbox publisher